apps/etterlatte-saksbehandling-ui/client/src/components/behandling/sidemeny/BehandlingSidemeny.tsx

```diff
@@ -138,7 +134,7 @@ export const BehandlingSidemeny = ({ behandling }: { behandling: IBehandlingRedu
       {isFailure(opprettSjekklisteResult) && erFoerstegangsbehandling && (
         <ApiErrorAlert>Opprettelsen av sjekkliste feilet</ApiErrorAlert>
       )}
-      {sjekklisteAktivert && !erRevurdering && (
+      {erFoerstegangsbehandling && (
```

Comment:
synes det er tydeligere å benytte dette,  og så blir sjekklista heller ikke aktivert for opphør